### PR TITLE
[Buttons] Add MWC buttons doc

### PR DESCRIPTION
**Pull request by arthurevans**
Take 1.

### CODE REVIEW

File: buttons-examples/Web-MWC/web-button-example.md [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+Text color | Primary color | N/A | None (can override `--mdc-theme-primary`)
+Typography | Default button text style | N/A | `--mdc-typography-font-family`, `--mdc-typography-button-*`
+
+For details on the available typography properties, see [Typography properties](../theming#typography-properties) in the Theming guide.
+
+#### Container attributes
+
+Design Attribute | Theme value | CSS property
+---|---|---
+Ripple color | Primary color at 16% opacity | None (can override `--mdc-theme-primary`)
```
Comment (dfreedm) [inline buttons-examples/Web-MWC/web-button-example.md:171]: Override color with `--mdc-ripple-color`. Override opacity with `--mdc-ripple-press-opacity`, `--mdc-ripple-focus-opacity` and `--mdc-ripple-press-opacity`

File: buttons-examples/Web-MWC/web-button-example.md [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
```
Comment (dfreedm) [inline buttons-examples/Web-MWC/web-button-example.md:238]: See above ripple comment

File: buttons-examples/Web-MWC/web-button-example.md [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+Typography | Default button text style | `--mdc-typography-font-family`, `--mdc-typography-button-*`
+
+For details on the available typography properties, see [Typography properties](../theming#typography-properties) in the Theming guide.
+
+#### Container attributes
+
+Design Attribute | Theme value | CSS Property
+---|---|---
+Container fill color| Primary color | None (can override `--mdc-theme-primary`)
+Ripple color | On-primary color at 32% opacity | None (can override `--mdc-theme-on-primary`)
```
Comment (dfreedm) [inline buttons-examples/Web-MWC/web-button-example.md:311]: This one is actually hardcoded right now, blocked on https://github.com/material-components/material-components-web-components/issues/1072

**Review by dfreedm — CHANGES_REQUESTED** [review]

File: buttons-examples/Web-MWC/web-button-example.md [hunk 4]
```diff
@@ -0,0 +1,457 @@
+<!--docs:
+title: "Buttons"
+layout: detail
+section: components
+excerpt: "Web Buttons"
+ide_version: "<cIDE name> <compatible IDE version and build number>"
+material_package_version: "<compatible Material platform package version number>"
+iconId:
+path: /:
```
Comment (mingjane-work) [inline buttons-examples/Web-MWC/web-button-example.md:9]: You need to change this to ensure that it's placed into the correct directory by the material.io import script:

path: /catalog/buttons

File: buttons-examples/Web-MWC/web-button-example.md [hunk 5]
```diff
@@ -0,0 +1,457 @@
...
+![Example of the four button types](assets/buttons_types.png)
+
+## Using buttons
+
+Before you can use a button, you will need to install to:
+
+* Install the button component.
+* Import the button definiton.
+
+See the [Quick start](https://github.com/material-components/material-components-web-components/blob/master/README.md#quick-start) for more information on using MWC components in your project.
```
Comment (mingjane-work) [inline buttons-examples/Web-MWC/web-button-example.md:33]: Is the Quick start eventually going to be part of the documentation imported into Material.io? If so, you may want to switch to a local link (talk to @jordanfine)

File: buttons-examples/Web-MWC/web-button-example.md [hunk 6]
```diff
@@ -0,0 +1,457 @@
...
+**`mwc-icon-button-toggle`**
+```bash
+npm install @material/mwc-icon-button-toggle
+```
+
+### Import the button definition
+
+You'll need to import the JavaScript definitions for the buttons you use.
+
+#### To import the button definitons
```
Comment (mingjane-work) [inline buttons-examples/Web-MWC/web-button-example.md:56]: I don't think this section header is necessary.

**Review by mingjane-work — CHANGES_REQUESTED** [review]

File: buttons-examples/Web-MWC/web-button-example.md (hunk 4 above)
Comment (arthurevans) [reply buttons-examples/Web-MWC/web-button-example.md:9]: 👍

**Pull request merged**